test.js

```diff
@@ -17,6 +17,12 @@ test('returns audio sources', async t => {
   }
 });
 
+test('returns available video codecs', t => {
+  const codecs = aperture.videoCodecs;
+  console.log('Video codecs:', codecs);
```

Comment:
Is console.log intentionally left here?

Reply:
Yes, just to manually verify everything is correct. I do it in the other test too.

Reply:
Why not just `t.deepEqual()`? There's probably a reason for it.

Reply:
The contents are not static. I could do deepEqual with some conditionals, but then I'm just reproducing the main code. This is just easier.

Reply:
👍